### ISSUE TITLE
Filter by new parking type field

### ISSUE THREAD

**Issue by emilyjablonski** [bloom-housing/bloom#5753]
Blocked by #5653

We need to be able to filter by the new parking type field if the feature's toggle is on.

[These are the designs](https://www.figma.com/design/qgL3N9ssyOMhpD5W4ULg7G/Filter-set--Accessibility?node-id=15-2793&p=f&t=ZHNSHyVTNotM0RcA-0) for the accessibility features, but we can also see the new parking type filter in there as well. It should be an optional set of checkboxes that mirrors the same partners set of options.

<img width="960" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeFNZREE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--5b05891bfb76767d56069c6f793dccc1db215446/Screenshot%202026-01-13%20at%204.38.22%E2%80%AFPM.png" alt="Screenshot 2026-01-13 at 4.38.22 PM.png" />

**Comment by emilyjablonski** [comment]
QA Notes
In Angelopolis, we should see a filter on the public site for parking type (https://bloom-angelopolis.netlify.app/listings?parkingType=onStreet)
We should not see this filter in Bloomington (https://bloom-public-seeds.netlify.app/listings)
